src/Symfony/Component/Asset/UrlPackage.php

```diff
@@ -123,6 +123,11 @@ protected function chooseBaseUrl($path)
         return (int) fmod(hexdec(substr(hash('sha256', $path), 0, 10)), \count($this->baseUrls));
     }
 
+    /**
+     * @param array $urls
+     *
+     * @return array
+     */
```

Comment:
Can be removed, we don't document private methods

Reply:
I've had a look at the [coding standards](https://symfony.com/doc/current/contributing/code/standards.html) page, private methods are also documented there. Just want to ask, as I want to follow the official standard ;-) otherwise someone might need to update the standards page? or am I missing something here?